### ISSUE TITLE
Feature list and can it create new address for new transactions?

### ISSUE THREAD

**Issue by BarbzYHOOL** [aspectron/kdx#57]
There is no feature list for the kaspa wallet

Can it create new address for new transactions (for privacy reasons)??

**Comment by aspect** [comment]
This software will be deprecated in the coming months with Kaspa NG. https://aspectron.org/en/projects/kaspa-ng.html

**Comment by BarbzYHOOL** [comment]
> This software will be deprecated in the coming months with Kaspa NG. https://aspectron.org/en/projects/kaspa-ng.html

ok thank you

Can the new wallet create new address for new transactions (for privacy reasons)??

**Comment by aspect** [comment]
Absolutely. We are also working on a feature set (not 100% complete yet)
where it will allow full privacy lockdown - anyone with full access to
computer will not be able to know about any wallet activity.

Manual new address creation is already supported and the wallet also
supports multiple accounts.

**Comment by BarbzYHOOL** [comment]
When it will be ready? I don't want to switch to KDX if it's gonna be replaced soon

**Comment by aspect** [comment]
It's fully functional, we are just waiting to see if anyone experiences any issues. I think in about 2 months it will be in a beta/release-candidate (but the software will be updated continuously for a while). It's stable as a node and so far the wallet doesn't seem to have any issues. That said the official timeline is about 2-3 months.

**Comment by BarbzYHOOL** [comment]
ok thank you
